Containers/nextcloud/entrypoint.sh

```diff
@@ -20,6 +20,61 @@ run_upgrade_if_needed_due_to_app_update() {
     fi
 }
 
+create_global_root_cert() {
```

Comment:
the certs should only be written out once at container start if possible and not multiple times

Reply:
done